### ISSUE TITLE
Migrate this repo into executor-python

### ISSUE THREAD

**Issue by roleyfoley** [hamlet-io/executor-cookiecutter#18]
## Expected Behaviour

The python executor and hamlet-cli maintain the cookiecutter requests for the base generate commands and specific repositories are setup for the more complicated repositories as library patterns 

## Current Behaviour

The hamlet cli requires this repository to be cloned locally in order to generate base templates

## Possible Solution

Move the base CMDB generation cookiecutter templates in to https://github.com/hamlet-io/executor-python 

## Context

The generate commands a quite handy within the hamlet-cli and the base product cmdb templates would be better suited as part of the hamlet cli which is the primary user interface for hamlet. Including these in this package reduces the dependencies required to setup and use hamlet

**Comment by RossMurr4y** [comment]
Does this need an ADR? There's probably a reasonable quantity of small documentation updates required across the Hamlet Deploy space that should be tracked on deprecation of this repo.

**Comment by roleyfoley** [comment]
What do we consider the trigger for an ADR? It doesn't necessarily change the architecture of hamlet and don't the docs mostly call out the use of the hamlet cli generate command rather than calling cookie cutter

**Comment by RossMurr4y** [comment]
Good point. Architecture is at a higher scope than the details of where the code is stored so isn't the right way to track this.

The docs should pretty much all reference the CLI, they'd just require a quick scan for outdated cookie cutter info.

**Comment by roleyfoley** [comment]
Need to update doco to align the new command names for generate of product cmdbs
